### PR TITLE
Add some missing functions

### PR DESCRIPTION
**Pull request by adrien-berchet**
Add some functions.
(this PR is a revival of #187 , #188 and #189)

### CODE REVIEW

File: geoalchemy2/functions.py [hunk 1]
```diff
@@ -139,10 +171,124 @@ def __init__(self, *args, **kwargs):
...
+     'counter-clockwise.'),
+
+    ('ST_ForceRHR', types.Geometry,
+     'Force the orientation of the vertices in a polygon to follow the '
+     'Right-Hand-Rule.'),
+
+    ('ST_ForceSFS', types.Geometry,
+     'Force the geometries to use SFS 1.1 geometry types only.'),
+
+    ('ST_M', types.Geometry,
```
Comment (elemoine) [inline geoalchemy2/functions.py]: Type is not correct.

**Comment by elemoine** [comment]
> While I was writing this PR, I wondered if it would be possible to just ask the DB which functions are returning a `Geometry` and to register these functions automatically, instead of enumerating all of them. Do you think it would be relevant? It seems possible with a query like this one:

Not sure I'd like to introduce something like this. There are a lot of PostGIS functions that don't return a Geometry, so why would we "favor" those that return geometries?

**Comment by adrien-berchet** [comment]
> > While I was writing this PR, I wondered if it would be possible to just ask the DB which functions are returning a `Geometry` and to register these functions automatically, instead of enumerating all of them. Do you think it would be relevant? It seems possible with a query like this one:
> 
> Not sure I'd like to introduce something like this. There are a lot of PostGIS functions that don't return a Geometry, so why would we "favor" those that return geometries?

You are right, this is not the good way of thinking. But maybe querying all the functions related to the PostGIS extension (I don't know the query for this, is it possible?) and getting their return type, or something like this?
I am just wondering but the number of PostGIS functions is not really an issue anyway.

**Comment by elemoine** [comment]
> You are right, this is not the good way of thinking. But maybe querying all the functions related to the PostGIS extension (I don't know the query for this, is it possible?) and getting their return type, or something like this?

It probably is possible, but that would be PostGIS-only.

**Comment by adrien-berchet** [comment]
> It probably is possible, but that would be PostGIS-only.

Ah yes, I always forget that geoalchemy2 also supports spatialite... :)

**Comment by elemoine** [comment]
@adrien-berchet I should have let you merge this. Hope this was complete on your side.

**Comment by adrien-berchet** [comment]
Indeed, you had the finger ready on the trigger for this one ;)
But no problem, it looked ok to me.

**Pull request merged**